testsuite/manualtests/test_conngen_interface.sli

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Because it requires a) the C++ version of libcsa, which is not yet publicly available and b) some extensions to our test harness (e.g. handling the installation and setting environment variables) that I don't dare implementing until a) is fulfilled.